src/test/java/uk/gov/companieshouse/controller/SuppressionControllerTest_POST.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.companieshouse.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.companieshouse.model.Suppression;
+import uk.gov.companieshouse.service.SuppressionService;
+
+import java.io.File;
+import java.util.function.Function;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
Rather then starting the full spring application context, we can isolate the web layer and instantiate only the controller we require by using the following annotation instead:

```suggestion
@WebMvcTest(SuppressionController.class).
```
https://spring.io/guides/gs/testing-web/